lighthouse-core/scripts/lantern/assert-master-lantern-values-unchanged.js

```diff
@@ -10,50 +10,62 @@
 
 const fs = require('fs');
 const path = require('path');
-const execSync = require('child_process').execSync;
 const constants = require('./constants');
 
+// @ts-ignore
+const chalk = require('chalk');
```

Comment:
chalk isn't in our deps, is that on purpose? (chalk also added its own types at some point, it looks like, though our transitive dep chalk is stuck on an old version)

Reply:
done